### PR TITLE
docs: Describe Polaris plugin and give overview for configuration checkers

### PR DESCRIPTION
**Pull request by danielpacak**
To preview the docs page:

```
make mkdocs-serve
```

And then open [http://localhost:8000](http://localhost:8000) in your web browser.

Signed-off-by: Daniel Pacak <pacak.daniel@gmail.com>

### CODE REVIEW

File: docs/integrations/config-checkers/index.md [hunk 1]
```diff
@@ -1 +1,22 @@
 # Configuration Checkers
+
+Starboard relies on a configuration checker to run variety of checks  on discovered Pods and controllers to make sure
```
Comment (itaysk) [inline docs/integrations/config-checkers/index.md:3]: "controllers" is not very descriptive in this context, maybe "workloads"?

File: docs/integrations/config-checkers/index.md [hunk 2]
```diff
@@ -1 +1,22 @@
...
+they are configured using best practices.
+
+You can choose a third party configuration checker, which is already integrated with Starboard, or implement your own
```
Comment (itaysk) [inline docs/integrations/config-checkers/index.md:6]: "third party ... which is already integrated" isn't very clear (almost conflicting). I suggest:
You can choose any of the included configuration checkers or implement your own...

File: docs/integrations/config-checkers/polaris.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# Polaris
+
+[Polaris] is the default configuration check used by Starboard. It runs a variety of checks to ensure that Kubernetes
```
Comment (itaysk) [inline docs/integrations/config-checkers/polaris.md:3]: check -> checker

**Review by itaysk — APPROVED** [review]
some very small comments

File: docs/integrations/config-checkers/index.md (hunk 1 above)
Comment (danielpacak) [reply docs/integrations/config-checkers/index.md]: Changed to workloads

File: docs/integrations/config-checkers/index.md (hunk 2 above)
Comment (danielpacak) [reply docs/integrations/config-checkers/index.md]: Applied suggestion

File: docs/integrations/config-checkers/polaris.md (hunk 3 above)
Comment (danielpacak) [reply docs/integrations/config-checkers/polaris.md]: Done

**Pull request merged**